manifests/init.pp

```diff
@@ -292,6 +292,26 @@
   if $pulpcore and !$pulpcore_mirror {
     include foreman
 
+    if $pulp {
+      include katello::pulp
+      $mongo_db_config = {
+        'mongo_db_name'         => $katello::pulp::mongodb_name,
+        'mongo_db_seeds'        => $katello::pulp::mongodb_seeds,
+        'mongo_db_username'     => $katello::pulp::mongodb_username,
+        'mongo_db_password'     => $katello::pulp::mongodb_password,
+        'mongo_db_replica_set'  => $katello::pulp::mongodb_replica_set,
+        'mongo_db_ssl'          => $katello::pulp::mongodb_ssl,
+        'mongo_db_ssl_keyfile'  => $katello::pulp::mongodb_ssl_keyfile,
+        'mongo_db_ssl_certfile' => $katello::pulp::mongodb_ssl_certfile,
+        'mongo_db_verify_ssl'   => $katello::pulp::mongodb_verify_ssl,
+        'mongo_db_ca_path'      => $katello::pulp::mongodb_ca_path,
+      }
+
+      class { 'pulpcore::plugin::migration':
+        * => $mongo_db_config,
```

Comment:
Is there an advantage to the `*` vs. just setting each parameter on the class itself?

Reply:
Nope, it's just an artifact of the previous approach I was using. It's no longer necessary, so I removed it.